infra/build/functions/target_experiment.py

```diff
@@ -81,6 +82,20 @@ def run_experiment(project_name, target_name, args, output_path,
   local_stacktrace_path = os.path.join(build.out, 'stacktrace/')
   fuzzer_args = ' '.join(args + [f'-artifact_prefix={local_artifact_path}'])
 
+  # Upload err.log.
+  if errlog_path:
+    compile_step_index = next((i for (i, d) in enumerate(steps)
+                               if '&& compile' in ' '.join(d['args'])), -1)
+    if compile_step_index == -1:
+      print('Cannot find compile step')
+    else:
+      # Insert the upload step right after compile step.
+      steps.insert(compile_step_index + 1, {
+          'name': 'gcr.io/cloud-builders/gsutil',
+          'args': ['-m', 'cp', local_err_path, errlog_path],
```

Comment:
nit: no need for `-m` given this is only one file? `-m` enables parallel transmission for multiple files.